Sources/PrimerSDK/Classes/Core/Payment Services/VaultService.swift

```diff
@@ -29,17 +29,8 @@ internal class VaultService: VaultServiceProtocol {
                 fetchVaultedPaymentMethods(clientToken: clientToken)
             }
             .done { paymentMethods in
-                AppState.current.selectedPaymentMethodId = paymentMethods.data.first?.id
                 state.paymentMethods = paymentMethods.data
-
-                if state.selectedPaymentMethodId == nil && !state.paymentMethods.isEmpty {
-                    guard let firstPaymentMethodToken = state.paymentMethods.first?.id else {
-                        seal.fulfill()
-                        return
-                    }
-                    state.selectedPaymentMethodId = firstPaymentMethodToken
-                }
-
```

Comment:
This seems like a decent change. What was going on here? Whats the difference between `AppState.current` and `state`?